robot/impl/local_robot_test.go

```diff
@@ -1912,232 +1910,6 @@ func TestConfigMethod(t *testing.T) {
 	test.That(t, actualCfg, test.ShouldResemble, &expectedCfg)
 }
 
```

Comment:
Just confirming, we want to be deleting TestReconnectRemote & TestReconnectRemoteChangeConfig?